hammeraj/onvif
# Implements SystemDateAndTime schema as well as Get and Set operations with tests

## What and Why?
Schema for SystemDateAndTime and implementation of both SetSystemDateAndTime and GetSystemDateAndTime
## Verification Steps
Make changes using `Onvif.Devices.SetSystemDateAndTime.request` to set and 
`Onvif.Devices.GetSystemDateAndTime.request` to check.
## Tested on
- UNIVIEW
- Homaxi
- HIKVISION
- Axis
